packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme_web.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+import 'dart:html';
+
+import 'package:flutter/widgets.dart';
+
+import '../../utils.dart';
+import '../logger/logger.dart';
+import 'ide_theme.dart';
+
+/// Load any IDE-supplied theming.
+IdeTheme getIdeTheme() {
+  final queryParams = _loadQueryParams();
+
+  final overrides = IdeTheme(
+    backgroundColor: _tryParseColor(queryParams['backgroundColor']),
+    foregroundColor: _tryParseColor(queryParams['foregroundColor']),
+    fontSize: _tryParseDouble(queryParams['fontSize']),
+  );
+
+  // If the environment has provided a background color, set it immediately
+  // to avoid a white page until the first Flutter frame is rendered.
+  if (overrides.backgroundColor != null) {
+    document.body.style.backgroundColor =
+        overrides.backgroundColor.value.toRadixString(16).padLeft(8, '0');
```

Comment:
don't you need a '#' at the start of the color string?

Reply:
Yep - apparently my testing of this was bad because the scaffold color was being set and I didn't notice the real bg was not.

While fixing this, I noticed there's a different in the position of alpha in CSS colors (#RRGGBBAA) and the order of Flutter's bytes (alpha is high bits) so I also handled this in case any editors pass through a 4/8 digit hex code (we would've previously interpreted this incorrectly).